src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedContainer.cs

```diff
@@ -15,6 +15,8 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     /// </summary>
     internal abstract class SynthesizedContainer : NamedTypeSymbol
     {
+        internal const string StructFieldCaptureName = "<struct-field-capture-name>";
```

Comment:
The field will need to be recognized by the EE to display the new local variables. As such it should be of the same format as other special generated names: ```<>``` {kind} -- see e.g. [http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp/Symbols/Synthesized/GeneratedNames.cs,377](http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp/Symbols/Synthesized/GeneratedNames.cs,377)

The kind needs to be listed in [http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp/Symbols/Synthesized/GeneratedNameKind.cs](http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp/Symbols/Synthesized/GeneratedNameKind.cs) enum.

---
In reply to: [134590720](https://github.com/dotnet/roslyn/pull/21664#discussion_r134590720) [](ancestors = 134590720)